forc-plugins/forc-client/src/ops/pkg_util.rs

```diff
@@ -0,0 +1,54 @@
+use std::{collections::HashMap, path::Path};
+
+use anyhow::Result;
+use forc_pkg::{self as pkg, manifest::ManifestFile, BuildOpts, BuildPlan};
+use pkg::{manifest::MemberManifestFiles, BuiltPackage, PackageManifestFile};
+
+fn built_pkgs(
+    build_opts: BuildOpts,
+    member_manifests: &MemberManifestFiles,
+) -> Result<HashMap<String, BuiltPackage>> {
+    let built = forc_pkg::build_with_options(build_opts)?;
+    match built {
+        pkg::Built::Package(built_pkg) => {
+            let pkg_name = member_manifests
+                .keys()
+                .next()
+                .expect("built package is missing");
+            Ok(std::iter::once((pkg_name.clone(), *built_pkg)).collect())
+        }
+        pkg::Built::Workspace(built_workspace) => Ok(built_workspace),
+    }
```

Comment:
Perhaps this could be a method on `Built`? E.g.
```rust
impl Built {
    pub fn into_members(self) -> HashMap<String, BuiltPackage> {
        // ...
    }
}
```